README.md

```diff
@@ -85,7 +85,7 @@ GameMode depends on `meson` for building and `systemd` for internal communicatio
 
 #### Ubuntu/Debian (you may also need `dbus-user-session`)
 ```bash
-apt install meson libsystemd-dev pkg-config ninja-build git libdbus-1-dev
+apt install meson libsystemd-dev pkg-config ninja-build git libdbus-1-dev libinih-dev
```

Comment:
Btw right now this actually isn't a real dependency, `libinih-dev` doesn't have a pkg-config entry in Debian (yet).